apps/docs/src/app/core/component-docs/datetime-picker/datetime-picker-docs.component.html

```diff
@@ -58,7 +58,10 @@
 <fd-docs-section-title [id]="'datetime-picker-formatting'" [componentName]="'datetime-picker'">
     Formatting
 </fd-docs-section-title>
-<description>Providing a custom format for the dates is possible through providing a service.</description>
+<description>
+    Providing a custom format for the dates is possible through providing a service.
+    Note that when providing a custom format, the <code>[placeholder]</code> input should be set to match the new format as well.
```

Comment:
> Note that when providing a custom format, the <code>[placeholder]</code> input should be set to match the new format as well.

Is is true with implementation changes introduced in this PR? Now when you set `[format]` input placeholder is automatically set to custom format.

Reply:
Ah I accidentally committed that.  It was from my first attempt at fixing this problem.  However the [format] input is actually not used for this purpose and due to DateTimeFormatParser/DateFormatParser it is not possible to assume the placeholder